src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,22 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  moviesFromServer: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ moviesFromServer }) => (
+  <div className="movies">
+    {moviesFromServer.map((movie: Movie) => (
+      <MovieCard
+        key={movie.imdbId}
+        title={movie.title}
+        description={movie.description}
+        imgUrl={movie.imgUrl}
+        imdbUrl={movie.imdbUrl}
```

Comment:
I think it would be better to pass the entire movie, and take properties that you need inside the component, so that you can avoid passing so many props and using Omit.